deps/oblib/src/lib/container/ob_vector.ipp

```diff
@@ -145,7 +145,7 @@ int ObVector<T, Allocator>::expand(int64_t size)
         copy(new_mem, mem_begin_, mem_end_);
       }
 
-      // deallocate old memeory
```

Comment:
error: patch failed: src/sql/rewrite/ob_transform_min_max.h
error: patch failed: src/sql/rewrite/ob_transform_semi_to_inner.cpp

remove the modification of this two file, there is conflict with internal codebase